HomeWorks/java/lesson4/Student.java

```diff
@@ -0,0 +1,11 @@
+package lesson4;
+
+public class Student {
+    int IdOfStudentTicket;
```

Comment:
Variable name should start with lowercase!
idOfStudentTicket
firstName
lastName